docs/source/community/pull_requests.rst

```diff
@@ -0,0 +1,360 @@
+   by messaging ``r2g`` (or something similar) in the PR conversation.
+#. Respectfully address any reviewer concern. Marking each one as resolved when
+   it has been addressed.
+#. If the PR has changed return to item 5.
+#. The last approving reviewer merges the PR after all CI workflows pass.
+
+.. note::
+
+   For PRs whose description requires more than a couple sentences. The author
+   should open a corresponding issue with the full description. The issue is
+   for tracking the design, scope, concerns, etc. that the PR should address.
+   The PR itself is for discussing how the PR author literally chose to
+   implement the feature, patch, etc.
+
+
+Review Process
+==============
+
+.. note::
+
+   GitHub allows reviewers to suggest changes. This is very useful when there's
+   a typo, formatting error, etc. Please use this feature rather than
+   writing comments like "should be capitalized".
+
+Reviewers of a PR are expected to:
+
+#. Understand what the PR is supposed to accomplish.
+#. If necessary, the reviewers should help the author refine the PR contents.
+
+   - Should the PR (and corresponding issue) be split into multiple issues/PRs?
+   - Did the author miss any obvious concerns?
+
+#. Keep an eye on the PR as it progresses. The frequency of "check-ins" should
+   be inversely proportional to the author's familiarity with the process,
+   *i.e.*, keep a closer eye on newer authors than seasoned veterans.
+#. Comment on the code when issues are spotted.
+
+   - Is the code using existing infrastructure to the extent possible?
+   - Is the code accruing technical debt?
+   - Is the formatting consistent? (Don't worry about formatting which CI will
+     fix)
+
+#. When the PR is marked as ready to go, complete a final pass through the code
+   flagging any potential issues.
+#. If issues arise work with the author to resolve them. Repeating the previous
+   step as necessary.
+#. If you are the last reviewer to approve a PR then merge it (assuming all
+   CI workflows have passed).
+
+Notes on PR Quality
+===================
+
+.. note::
+
+   The contents of this section are written assuming a 1.0 has occurred. We
+   admittedly have not lived up to the lofty standards of this section and
+   part of getting to a 1.0 is making sure existing code meets or exceeds
+   these standards.
+
+.. note::
+
+   Occasional contributors from outside the project are not the target of this
+   section. This section is targeting developers who are part of the team.
+
+The checklist on the PR template is admittedly asking a lot of the author.
+NWChemEx is designed to be a modular code. The vast majority of electronic
+structure development occurs in modules. Each of these modules are disjoint,
+and can be separately hosted. In turn, you can (and should) go through the
+"just get something working" phase outside of the NWChemEx repositories.
+
+By making a PR which will add a module into an NWX repository you are saying the
```

Comment:
How do we synch this with wanting people to write draft PRs to have discussions while the modules are being developed?

Reply:
I think I addressed this, but let me know if I didn't.